rllib/offline/offline_evaluator.py

```diff
@@ -48,3 +55,21 @@ def train(self, batch: SampleBatchType, **kwargs) -> Dict[str, Any]:
             Any optional metrics to return from the evaluator
         """
         return {}
+
+    @ExperimentalAPI
+    def estimate_on_dataset(
+        self, 
+        dataset: Dataset, 
+        *, 
+        n_parallelism: int = os.cpu_count(),
+    ) -> Dict[str, Any]:
+
+        """Calculates the estmiate of the metrics based on the given offline dataset. 
```

Comment:
Describe more here how this happens: We run once(?) through the entire dataset?

Reply:
done